### DISCUSSION TITLE
View not loading on Vapor

### DISCUSSION THREAD

**Discussion by claybitner** [njxqlus/filament-relation-manager-component#9] [Q&A] [answered]
I've been bashing my head against a brick wall with this for a few days now and can't understand why... This plugin works great locally, but when I push it to my AWS server through Laravel Vapor, it can't seem to find the Views and it's throwing 500 errors. Any idea why this might be happening?

**Comment by claybitner** [comment, marked as answer]
Fixed it by publishing the views and renaming `Infolists` to `infolists` - needs a lowercase "i" to work on Vapor. Would be nice to fix that in the repo.

**Reply by njxqlus** [reply]
Please, make a pull request
